tests/handlers/test_typing.py

```diff
@@ -62,7 +63,11 @@ def _make_edu_transaction_json(edu_type: str, content: JsonDict) -> bytes:
 
 
 class TypingNotificationsTestCase(unittest.HomeserverTestCase):
-    def make_homeserver(self, reactor: MemoryReactor, clock: Clock) -> HomeServer:
+    def make_homeserver(
+        self,
+        reactor: ThreadedMemoryReactorClock,
```

Comment:
Why the change from `MemoryReactor`? It isn't clear to me what's wrong with that?

Reply:
This seems to be because we use `pump()` which is defined on `MemoryReactorClock`. And turns out that the super-class *does* use `ThreadedMemoryReactorClock`.